templates/hooks/subnetgroup/sdk_create_post_set_output.go.tpl

```diff
@@ -0,0 +1,16 @@
+var resourceSyncedCondition *ackv1alpha1.Condition = nil
```

Comment:
You should not be needing Custom hook for setting Synced condition. If there are no errors and this condition will set to true by default at the end of reconciliation loop. If you want to control it then you can use SyncedConfig inside generator.yaml.